data/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
+  "project_info": {
+    "project_number": "737392897175",
+    "project_id": "mowid-764f2",
+    "storage_bucket": "mowid-764f2.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:737392897175:android:bceab03869cef60f9e0dd2",
```

Comment:
Since this is a public repo - I guess this file shouln't be added to the repo at all